checker/bin/javac_maven

```diff
@@ -13,3 +13,8 @@ case `uname -s` in
 esac
 
 eval "java" "-jar" "${mydir}"/../dist/checker.jar -processorpath "${mydir}"/../dist/checker.jar "\@argfile" "$@"
+
+echo
+echo
+echo NOTE: The javac_maven script is deprecated and will be removed in the December 1, 2015 release.
```

Comment:
Here and below, I would just say "in December 2015".  That's all that users care about; the extra information is likely to just be confusing to them.